scripts/lintcommit.lua

```diff
@@ -0,0 +1,149 @@
+local function commit_message_is_ok(commit_message)
+  local commit_split = vim.split(commit_message, ":")
+
+  -- Return true if the type is vim-patch since most of the normal rules don't
+  -- apply.
+  if commit_split[1] == "vim-patch" then
+    return true
+  end
+
+  -- Check that message isn't too long.
+  if commit_message:len() > 80 then
+    vim.cmd("verbose echo 'Commit message is too long, a maximum of 80 characters is allowed. Aborting...'")
+    vim.cmd("verbose echo ''")
+    return false
+  end
+
+  -- Return false if no colons are detected.
+  if vim.tbl_count(commit_split) < 2 then
+    vim.cmd("verbose echo 'Commit message does not include colons. Aborting...'")
+    vim.cmd("verbose echo ''")
```

Comment:
Maybe wrap this two common statements in a function `output(msg)`  also the empty echo is it for a new line? Can you add "\n" at the end of the first echo?. 

Also can do multiline 
```
vim.cmd([[
echo this
echo that]])
```

Reply:
Yep, that was exactly why. I got the \n to work, but only if I used exactly [[]] and "". Thanks for the tip.